components/Header/index.js

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+import { useRouter } from "next/router";
+
+export default function Header() {
+  const { pathname } = useRouter();
+  console.log(pathname);
+
+  if (!pathname.startsWith("/movies"))
+    return (
+      <>
+        <StyledHeader>
+          <p>Just Find It</p>
+        </StyledHeader>
+      </>
+    );
+}
```

Comment:
👉 The reason why we outsource parts as stand-alone components is to make them reusable. For this reason, they should be as dumb as possible.

```suggestion
export default function Header({headerTitle, variant}) {
    return (
        <StyledHeader>
          <h1>{headerTitle}</h1>
        </StyledHeader>
    );
}
```